quickstart/101-vm-cluster-windows/readme.md

```diff
@@ -0,0 +1,23 @@
+# Azure Windows VM cluster
+
+This template deploys a Windows VM cluster on Azure.
+
+## Terraform resource types
+
+- [random_pet](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/pet)
+- [azurerm_resource_group](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group)
+- [random_string](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/string)
+- [random_password](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/password)
+- [windows_server](https://registry.terraform.io/modules/Azure/compute/azurerm)
```

Comment:
Should modules be in their own section? I think it is a bit confusing to have moules listed under resource types.